homeassistant/components/systemmonitor/util.py

```diff
@@ -78,6 +78,7 @@ def read_cpu_temperature(temps: dict[str, list[shwtemp]]) -> float | None:
     entry: shwtemp
 
     _LOGGER.debug("CPU Temperatures: %s", temps)
+    # {'acpitz': [shwtemp(label='', current=47.0, high=103.0, critical=103.0)] }
```

Comment:
If you want to keep this example, please add a comment with context. If not, remove it.